src/Typed/README.md

```diff
@@ -0,0 +1,84 @@
+# Symfony UX Typed
+
+Symfony UX Typed is a Symfony bundle integrating [Typed](https://github.com/mattboldt/typed.js/blob/master/README.md) in
+Symfony applications. It is part of [the Symfony UX initiative](https://symfony.com/ux).
+
+Typed is a complete and easy to use animated typed texts.
+Just enter the strings you want to see typed, and it goes live without complexity.
+
+![Typed in action](Animation.gif)
+
+## Installation
+
+Symfony UX Typed does not use any PHP and can be installed with any PHP/Symfony version.
+
+You can install this bundle using Composer and Symfony Flex:
+
+```sh
+composer require symfony/ux-typed
+
+# Don't forget to install the JavaScript dependencies as well and compile
+yarn install --force
+yarn encore dev
+```
+
+Also make sure you have at least version 2.0 of [@symfony/stimulus-bridge](https://github.com/symfony/stimulus-bridge)
+in your `package.json` file.
```

Comment:
You can remove this - it's been long enough now since 2.0 (and 3.0) was released :)